### PR TITLE
Release/7.2.5

### PR DESCRIPTION
**Pull request by nwang92**

### CODE REVIEW

**Review by lephino — APPROVED** [review]

**Review by lephino — DISMISSED** [review]

**Review by bb03 — DISMISSED** [review]

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,114 +1,144 @@
...
+Have ideas on improvements? See something that needs work? While the community encourages everyone to contribute code, it is also appreciated when someone reports an issue. Please report any issues or bugs you find through [GitHub's issue tracker](https://github.com/splunk/docker-splunk/issues). 
+
+If you are reporting a bug, please include:
+* Your operating system name and version
+* Any details about your local setup that might be helpful in troubleshooting (ex. Python interpreter version, Ansible version, etc.)
+* Detailed steps to reproduce the bug
+
+We'd also like to hear about your propositions and suggestions. Feel free to submit them as issues and:
+* Explain in detail how they should work
+* Keep the scope as narrow as possible -t his will make it easier to implement
```
Comment (aegershman) [inline docs/CONTRIBUTING.md]: minor, typo on `-t his`

File: docs/INTRODUCTION.md [hunk 2]
```diff
@@ -1,30 +1,24 @@
...
-We are sorry for any inconvenience this may cause. 
 
+Universal Forwarders and standalone instances were being brought online at a rapid pace, which introduced a new level of complexity into the enterprise environment. In 2018, a new container image was created to improve the flexibility with which Splunk Enterprise could be operated in larger and more dynamic environments. Splunk's new container can now start with a small environment and grow with the deployment. This however has caused a divergence from the open-source community edition of the Splunk Enterprise container. 
+
+As a result, containers for Splunk Enterprise versions prior to 7.1 can not  be used with, or in conjunction with, this new version as it is not backward compatible. We are also unable to support version updates from any prior container to the current version released with Splunk Enterprise and Splunk Universal Forwarder 7.2, as the older versions are not forward compatible. We are sorry for any inconvenience this may cause.
```
Comment (aegershman) [inline docs/INTRODUCTION.md]: painfully obnoxiously minor, but unnecessary double spacing in `... not  be...`

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (nwang92) [reply docs/CONTRIBUTING.md]: 👍

File: docs/INTRODUCTION.md (hunk 2 above)
Comment (nwang92) [reply docs/INTRODUCTION.md]: Done, thanks!

**Review by arctan5x — APPROVED** [review]

**Review by bb03 — APPROVED** [review]

**Pull request merged**